src/doc/generate-documentation.lisp

```diff
@@ -25,9 +25,7 @@
             (:include documentation-entry))
   (type             (util:required 'type)             :type tc:ty                     :read-only t)
   (tyvars           (util:required 'tyvars)           :type tc:tyvar-list             :read-only t)
-  (fields           (util:required 'fields)           :type util:string-list          :read-only t)
-  (field-docstrings (util:required 'field-docstrings) :type hash-table                :read-only t)
-  (field-tys        (util:required 'field-tys)        :type hash-table                :read-only t)
+  (fields           (util:required 'fields)           :type tc:struct-field-list      :read-only t)
```

Comment:
Have you tried building documentation locally to make sure the docstrings are still transmitted?

Reply:
Hm, nope, good point. It built and tested clean, so I pushed and immediately went for a cheerful walk around the block.

I've now confirmed that docstrings are still propagated.